quarkchain/cluster/grpc_client.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import argparse
+import grpc
+
+from quarkchain.generated import grpc_pb2
```

Comment:
one line should be enough

`from quarkchain.generated import grpc_pb2, grpc_pb2_grpc`